contracts/Registry.vy

```diff
@@ -23,6 +23,11 @@ releases: public(HashMap[uint256, address])
 nextDeployment: public(HashMap[address, uint256])
 vaults: public(HashMap[address, HashMap[uint256, address]])
 
+# Maintain a list of endorsed vault tokens
+tokensList: public(address[65536])
+tokensMap: public(HashMap[address, bool])
+tokensCount: public(uint256)
```

Comment:
refactor to:
```python
# index of token added => token address
tokens: public(HashMap[uint256, address])
# len(tokens)
numTokens: public(uint256)
# inclusion check for token
isRegistered: public(HashMap[address, bool])
```

Reply:
I added the recommended fix and updated the unit tests as well